photon-ml/src/main/scala/com/linkedin/photon/ml/util/DefaultIndexMap.scala

```diff
@@ -36,7 +41,9 @@ class DefaultIndexMap(@transient val featureNameToIdMap: Map[String, Int]) exten
     _idToNameMap.get(idx)
   }
 
-  override def getIndex(name: String): Int = featureNameToIdMap.getOrElse(name, IndexMap.NULL_KEY)
+  override def getIndex(name: String): Int = {
+    featureNameToIdMap.getOrElse(name, IndexMap.NULL_KEY)
+  }
```

Comment:
It appears to my IDE that it does not fit in one line (length of this line == 99).